tooling/cli/src/migrate/migrations/v2_rc.rs

```diff
@@ -0,0 +1,195 @@
+// Copyright 2019-2024 Tauri Programme within The Commons Conservancy
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-License-Identifier: MIT
+
+use crate::{
+  helpers::{
+    app_paths::{app_dir, tauri_dir},
+    npm::PackageManager,
+  },
+  interface::rust::manifest::{read_manifest, serialize_manifest},
+  Result,
+};
+
+use std::{fs::read_to_string, path::Path};
+
+use anyhow::Context;
+use toml_edit::{Document, Item, Table, TableLike, Value};
+
+pub fn run() -> Result<()> {
+  let app_dir = app_dir();
+  let tauri_dir = tauri_dir();
+
+  let manifest_path = tauri_dir.join("Cargo.toml");
+  let (mut manifest, _) = read_manifest(&manifest_path)?;
+  migrate_manifest(&mut manifest)?;
+
+  migrate_permissions(&tauri_dir)?;
+
+  migrate_npm_dependencies(app_dir)?;
+
+  std::fs::write(&manifest_path, serialize_manifest(&manifest))
+    .context("failed to rewrite Cargo manifest")?;
+
+  Ok(())
+}
+
+fn migrate_npm_dependencies(app_dir: &Path) -> Result<()> {
+  let pm = PackageManager::from_project(app_dir)
+    .into_iter()
+    .next()
+    .unwrap_or(PackageManager::Npm);
+
+  let mut install_deps = Vec::new();
+  for pkg in [
+    "@tauri-apps/cli",
+    "@tauri-apps/api",
+    "@tauri-apps/plugin-authenticator",
+    "@tauri-apps/plugin-autostart",
+    "@tauri-apps/plugin-barcode-scanner",
+    "@tauri-apps/plugin-biometric",
+    "@tauri-apps/plugin-cli",
+    "@tauri-apps/plugin-clipboard-manager",
+    "@tauri-apps/plugin-deep-link",
+    "@tauri-apps/plugin-dialog",
+    "@tauri-apps/plugin-fs",
+    "@tauri-apps/plugin-global-shortcut",
+    "@tauri-apps/plugin-http",
+    "@tauri-apps/plugin-log",
+    "@tauri-apps/plugin-nfc",
+    "@tauri-apps/plugin-notification",
+    "@tauri-apps/plugin-os",
+    "@tauri-apps/plugin-positioner",
+    "@tauri-apps/plugin-process",
+    "@tauri-apps/plugin-shell",
+    "@tauri-apps/plugin-sql",
+    "@tauri-apps/plugin-store",
+    "@tauri-apps/plugin-stronghold",
+    "@tauri-apps/plugin-updater",
+    "@tauri-apps/plugin-upload",
+    "@tauri-apps/plugin-websocket",
+    "@tauri-apps/plugin-window-state",
+  ] {
+    let version = pm
+      .current_package_version(pkg, app_dir)
+      .unwrap_or_default()
+      .unwrap_or_default();
+    if version.starts_with("1") {
+      install_deps.push(format!("{pkg}@^2.0.0-rc.0"));
+    }
+  }
+
+  if !install_deps.is_empty() {
+    pm.install(&install_deps, app_dir)?;
+  }
+
+  Ok(())
+}
+
+fn migrate_permissions(tauri_dir: &Path) -> Result<()> {
+  let core_plugins = [
+    "app",
+    "event",
+    "image",
+    "menu",
+    "resources",
+    "tray",
+    "webview",
+    "window",
```

Comment:
I think that is missing the core pseudo `path` plugin

Reply:
of course, i got it from the other PR 😂